src/packages/@ncigdc/modern_components/BiospecimenCard/BioTree/index.js

```diff
@@ -204,25 +203,12 @@ const EnhancedBranch = compose(
         type: 'TOTAL_NODES',
       });
     },
-    componentWillMount(): void {
-      const {
-        setEmitterToken,
-        setExpanded,
-      } = this.props;
-      const token = Emitter.addListener('expand', toExpand => {
-        setExpanded(toExpand);
-      });
-
-      setEmitterToken(token);
-    },
```

Comment:
No sure why this part is removed.

Reply:
it breaks the app, and isn't necessary after the BioTree state management reimplementation.